x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/metric_item_icon.tsx

```diff
@@ -71,7 +71,8 @@ export const MetricItemIcon = ({
   });
   const euiShadow = useEuiShadow('s');
 
-  const testTime = useFormatTestRunAt(timestamp);
+  const formatter = useDateFormat();
+  const testTime = formatter(timestamp);
```

Comment:
can these two operations be combined in the hook? i see it being used in multi places..

Reply:
My motivation for returning a function vs. simply calling the hook was for cases where `formatter` needs to be called multiple times deeper in the component. But, there don't seem to be any instances of this, so if you prefer I can modify the API for the hook.

Reply:
that seems like a valid use case, so i am fine.

Reply:
Yeah I actually just looked into this a second ago and there are some cases where we're formatting in conditional blocks. I can make a "convenience" version that will perform the formatting immediately though. I might do that as a follow-up since this is pretty much good to go and has been open for so long.